connectorx-python/src/arrow2.rs

```diff
@@ -20,7 +38,209 @@ pub fn write_arrow<'a>(
     origin_query: Option<String>,
     queries: &[CXQuery<String>],
 ) -> &'a PyAny {
-    let destination = get_arrow2(source_conn, origin_query, queries)?;
+    let mut destination = ArrowDestination::new();
```

Comment:
This part of the code is relocated [here](https://github.com/sfu-db/connector-x/blob/main/connectorx/src/get_arrow2.rs). Can copy the newly added simple protocol related to the new location and keep this file not modified. The same for `arrow.rs`.